packages/labs/context/src/lib/decorators/consume.ts

```diff
@@ -48,7 +48,7 @@ export function consume<ValueType>({
   context: ContextKey<unknown, ValueType>;
   subscribe?: boolean;
 }): <K extends PropertyKey>(
-  protoOrDescriptor: ReactiveElement & Record<K, ValueType>,
+  protoOrDescriptor: ReactiveElement,
```

Comment:
Seems like the generics `ValueType` and `K` are no longer used, remove?

Reply:
I guess this just means we remove what little type safety existed... but I guess it's just possible to express?

Does `Partial<Record<K, ValueType>>` help? That's sorta like _if_ it's there then it has to have the right type?

Reply:
Yeah, that's better. I added a test that we get the error too.